phalcon/Session/Adapter/AbstractAdapter.zep

```diff
@@ -53,7 +53,10 @@ abstract class AbstractAdapter implements SessionHandlerInterface
      */
     public function read(var id) -> string
     {
-        return this->adapter->get(id);
+        var data;
+        let data = this->adapter->get(id);
+
+        return is_null(data) ? "" : data;
```

Comment:
could you please replace this to the language instruction instead of function
```suggestion
        return null === data ? "" : data;
```

or shorter

```diff
-        return is_null(data) ? "" : data;
+        return (string) data;
```